src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -111,10 +265,36 @@ onMounted(() => {
       layout: {
         "icon-allow-overlap": true,
         "icon-image": "buoy-marker",
-        "icon-offset": [0.5, -7], // optically centered
+        "icon-size": 1.2,
+        "icon-offset": [0.8, -4], // optically centered
       },
     });
 
+    for (var b = 0; b < windSpeedBins.length - 1; b++) {
+      map.value.addLayer({
+        id: "svgfile" + b,
+        type: "symbol",
+        filter: [
+          "all",
+          [">=", "windSpeed", windSpeedBins[b]],
+          ["<", "windSpeed", windSpeedBins[b + 1]],
+        ],
+        source: "points", // points.filter(annotatedwind)?
+        layout: {
+          "icon-allow-overlap": true,
+          "icon-image": `wind-speed-${b + 1}`, //mapped value of wind
+          "icon-offset": [-100, -40], // optically centered
+          "icon-size": 0.3,
+          "icon-rotate": {
+            property: "windDirection",
+            stops: [
+              [0, 90],
+              [270, 360],
+            ],
+          },
```

Comment:
Question about the rotate! So, if the two buoys have the same windspeed and different winddirections will the different wind directions show? 

At the moment,  I'm realizing that the for loop at 273 goes through each wind direction bin. But I'm wondering if instead it should be looping through each buoy?

Reply:
it does rotate for wind indicators at each buoy, I believe through the "points"